src/app/(sidebar)/account/create/page.tsx

```diff
@@ -46,27 +92,61 @@ export default function CreateAccount() {
             <Button size="md" variant="secondary" onClick={generateKeypair}>
```

Comment:
The previous state wasn't reset when clicking "Generate keypair` after successfully funding an account.

Should we also reset the state when user leaves this page? It feels a bit off to see the generated keypair when returning to the page.

Reply:
i think that makes sense if they save the keys to localStorage. the current lab currently saves the state. i personally prefer what you suggest since the code would be much simpler. maybe i'll create a task to add this once we enable `save keypairs` functionality

Reply:
will add resetting before generating keypair